### DISCUSSION TITLE
Terminal lock readonly

### DISCUSSION THREAD

**Discussion by xxcosita3czxx** [cockpit-project/cockpit#20720] [Q&A] [answered]
Page: 
Terminal

I found out that limited access mode is good for all pages except terminal, because anyone with basic knowledge could do damage with terminal only, so i would like to ask if its possible to make terminal readonly in limited access mode

**Comment by martinpitt** [comment, marked as answer]
You can do exactly as much damage in a Cockpit session as in an ssh session. You can't defend against malicious users with Cockpit, as it's a standard Linux session. You can prevent "accidents" by hiding the terminal page if you want to, see  https://cockpit-project.org/guide/latest/packages.html#package-manifest-override

**Reply by xxcosita3czxx** [reply]
Well can there be atleast way to lock the terminal when i am away from the computer? There could be scenario where i had running build and i cant just log out, i would lose access to that session and wont know if the build finishes or not.

**Reply by martinpitt** [reply]
You can use `screen`, `tmux`, `byobu` etc. in the terminal and then re-attach. Or better, when you are away from your computer lock your screen -- anyone who can get access to your desktop session with the browser also has access to all your connected SSH sessions, regardless of what the "Terminal" page does.

**Reply by xxcosita3czxx** [reply]
Okay :/
